README.md

```diff
@@ -33,6 +33,45 @@ of feedback on any [communication channels](#communication-channels)!
 
 ## Getting Started
 
+### Quick Start using Python
+
+Install from IREE's [releases](https://github.com/google/iree/releases) into a
+[venv](https://docs.python.org/3/library/venv.html):
+
+```bash
+$ python -m venv ~/.scratch
+$ source ~/.scratch/bin/activate
+$ python -m pip install iree-compiler-snapshot iree-runtime-snapshot -f https://github.com/google/iree/releases
+```
+
+Compile and run a program:
+
+```python
+import numpy as np
```

Comment:
Can we embed/link to the colab instead? This is going to bitrot *extremely* fast and always be out of date. It's also a big wall of deep internals that we never want anyone to use, so I'd prefer not to have it at all until we have an API we support :)

(I'm dealing with a bunch of similar bitrot in docs/ that is caused by people putting code in there and want to avoid introducing more)

Reply:
> ...I'd prefer not to have it at all until we have an API we support...
(to clarify, colab = good, including in readme = sketchy)

Reply:
SG, that satisfies my concerns as well. I'll rework a bit to add notes in the root README and elaborate on the docs in https://github.com/google/iree/blob/main/colab/README.md.

Reply:
Iterated a bit, may do more tomorrow (especially if the TF python releases go through successfully)